src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.types.ts

```diff
@@ -261,7 +261,7 @@ export interface IADT3DSceneBuilderElementsProps {
         isSelected: boolean
     ) => void;
     clearSelectedElements?: () => void;
-    onCreateBehaviorClick?: () => void;
```

Comment:
the arg here would be `initialBehaviorName`, it might get filled with the search term value but as far as this function is concerned, it should be provided to fill in the name.